src/app/pages/product/product-page.component.html

```diff
@@ -5,6 +5,13 @@
     <ng-container *ngIf="product$ | async as product">
       <ish-breadcrumb></ish-breadcrumb>
 
+      <!-- ViewContext usage example
+      <ish-content-viewcontext
+        viewContextId="vc_product_detail"
+        [callParameters]="{ Product: product.sku, Category: (category$ | async)?.categoryRef }"
+      ></ish-content-viewcontext>
+      -->
```

Comment:
Do we not plan to just set those active for the demo deployment? I'd rather not have dead code...

Reply:
Also, please subscribe only once to `category$` in the template, when activating this.

Reply:
Any good idea how to do that? I did not find a way besides using `*ngIf` which I did not want to use since the `category$` is more or less an optional value?

Reply:
Ah, yes, this is a problem. Maybe it is finally time to properly import https://ngrx.io/guide/component/let

Reply:
We cannot activate the ViewContexts as long as they are not system managed in ICM meaning there are probably no fitting ViewContexts for the given 'viewContextId'. So I will remove the dead code for now an just leave the documentation example in `content-viewcontext.component.ts`.

Reply:
This "solves" the category problem too for now. ;)